docs/octopus/octopus_0.7.4.md

```diff
@@ -0,0 +1,42 @@
+## CWL and Docker for Running Octopus
+
+## Version of tools in [docker image](https://hub.docker.com/r/biocontainers/bedtools)
```

Comment:
Can we remove the bedtools part and point to the image that you got octopus from.